contracts/utils/FunderLite.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+// Funder is nice but it may be a bit overkill for funding only two wallets
+contract FunderLite {
+    // Reading these three immutable values is cheaper than reading `root` from
+    // storage
+    address public immutable recipient;
+    uint256 public immutable lowThreshold;
+    uint256 public immutable highThreshold;
+
+    // Deployed normally (is not cloned) to reduce gas cost of funding calls
+    // Do not validate variables to reduce bytecode
+    constructor(
+        address _recipient,
+        uint256 _lowThreshold,
+        uint256 _highThreshold
+    ) {
+        recipient = _recipient;
+        lowThreshold = _lowThreshold;
+        highThreshold = _highThreshold;
+    }
+
+    // Anyone can send ETH to this contract, omitted withdrawal implementation
+    // to reduce bytecode
+    receive() external payable {}
+
+    // No Merkle proofs. Still allows reward mechanism to be built on top.
+    function fund() external {
+        uint256 recipientBalance = recipient.balance;
+        require(recipientBalance <= lowThreshold, "Balance not low enough");
+        uint256 topUpAmount = highThreshold - recipientBalance <
+            address(this).balance
+            ? highThreshold - recipientBalance
+            : address(this).balance;
```

Comment:
`unchecked` is cheaper because it omits that `require` so I don't think so. Keeping `highThreshold - recipientBalance` and `address(this).balance` in variables would improve things though. I wanted to keep things readable for now.

Reply:
Done in https://github.com/api3dao/airnode-protocol-v1/pull/129/commits/d88a0669a86e4afe34972582ade3164b53a9a58c